local-container-endpoints/handlers/credentials_handler.go

```diff
@@ -233,3 +244,28 @@ func (service *CredentialService) isCurrentSessionTemporary() bool {
 	}
 	return false
 }
+
+// Return an expiration date a set point in the future. error is currently
+// always nil (we gracefully fail back to the 12.5 minute default), but we
+// reserve it for future use in case there are valid reasons to error out.
+func getSharedTokenExpiration() (time.Time, error) {
+	durationStr := utils.GetValue(fmt.Sprintf("%ds", config.DefaultSharedTokenExpiration), config.SharedTokenExpirationVar)
+	duration, err := time.ParseDuration(durationStr)
+
+	if err != nil {
+		// If they didn't provide a unit, try to parse this as seconds.
+		durationSeconds, err := strconv.ParseInt(durationStr, 0, 64)
+		if err != nil {
+			durationSeconds = config.DefaultSharedTokenExpiration
```

Comment:
I think it might be good to have a log statement here like `"Could not parse SHARED_TOKEN_EXPIRATION value, defaulting to %d", config.DefaultSharedTokenExpiration`

Reply:
Will do!